docker/push_image.sh

```diff
@@ -1,38 +0,0 @@
-#!/bin/bash
```

Comment:
I don't understand the need to remove this entire file, it is useful for manually pushing an image to ACR

Reply:
it is not used

Reply:
As said, it can be useful for local development and for quickly pushing an image to a container registry without having to use the release pipeline. Unless there's a specific reason to remove it, I would not delete it.